### PR TITLE
Updated Building Dependencies

### PR DESCRIPTION
**Pull request by reflexran**
Adding Section of dependencies that may be required to build vinegar, see vinegar-dev in the discord for more info

### CODE REVIEW

**Comment by lunarlettuce** [comment]
you may need some udev/usblibs for controller support

**Comment by reflexran** [comment]
> you may need some udev/usblibs for controller support

These are just the Dependencies needed by Gioui, can you please specify which dependencies are needed for controller support?

**Review by apprehensions — CHANGES_REQUESTED** [review]
Please do not copy and paste the already provided gioui documentation, tell the user to visit it for dependency installation directly.

File: src/Installation/guides/source.md [hunk 1]
```diff
@@ -10,10 +10,7 @@ Before you can install Vinegar, you will have to install Wine first. You can use
 It is recommended to ensure that you have installed Wine 8.3 or above on your system, as Roblox requires a minimum version of 8.0 and may encounter stability issues under 8.3. 
 
 ## Building Dependencies
-If on Ubuntu or Fedora you may need to install the following dependencies:
-
-- Ubuntu: `apt install pkg-config libwayland-dev libx11-dev libx11-xcb-dev libxkbcommon-x11-dev libgles2-mesa-dev libegl1-mesa-dev libffi-dev libxcursor-dev libvulkan-dev`
-- Fedora: `dnf install pkg-config wayland-devel libX11-devel libxkbcommon-x11-devel mesa-libGLES-devel mesa-libEGL-devel libXcursor-devel vulkan-headers`
+If on Ubuntu, Fedora or Nixos you may need to install the dependencies needed by Gioui, Those can be found [here](https://gioui.org/doc/install/linux)
```
Comment (apprehensions) [inline src/Installation/guides/source.md:13]: Like i said, dont copy paste this or the mentioned distributions.
```suggestion
You will need to install the dependencies needed by Gio. The documentation can be found [here](https://gioui.org/doc/install/linux)
You can disable the splash window provided by Vinegar (Which is implemented with Gio) by appending the `nosplash` build tag to the build parameters (`VINEGAR_GOFLAGS=--tags=nosplash`)
```

**Pull request merged**